pkg/svcat/service-catalog/parameters.go

```diff
@@ -24,9 +24,9 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
 )
 
-// BuildParameters converts a map of variable assignments to a byte encoded json document,
+// BuildGenericParameters converts a gneric map to o a byte encoded json document,
```

Comment:
We have a mismatch between the godoc name and the function name here. Also "gneric" is a typo, along with "to o a byte".

Reply:
Argh yeah i noticed that as I was updating the flag. Fixed in this update! Sorry about that.

Reply:
![happy-little-accidents](https://user-images.githubusercontent.com/1368985/36927127-180661c8-1e41-11e8-888c-46dbe885c4d7.jpg)